frontend/app/src/premium/register-components.ts

```diff
@@ -130,6 +127,8 @@ export function registerComponents() {
   // Version 18 - 1.25
   Vue.component('MenuTooltipButton', MenuTooltipButton);
   Vue.component('GraphTooltipWrapper', GraphTooltipWrapper);
+  // Version 19 - 1.26
+  Vue.component('LpPoolIcon', LpPoolIcon);
```

Comment:
Btw this version code `19` needs to also be updated in [`setup-interface.ts`](https://github.com/rotki/rotki/blob/0482b4e34bd67c3a2c5964ab58dd359ea3c996d8/frontend/app/src/premium/setup-interface.ts#L101)

And also we need to bump the minimum somewhere in components